### PR TITLE
AB#6903 Performance improvements

### PR DESCRIPTION
**Pull request by adoprog**

### CODE REVIEW

**Review by mintsitecore — APPROVED** [review]
looks good

File: Website/src/rendering/src/assets/css/components/Page Content/ThreeColumnsSection.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .section--3-col {
   @apply bg-cover bg-center bg-no-repeat;
 
-  background-image: url(../../../img/background-pattern-light.svg);
+  background-image: url("data:image/svg+xml,%3Csvg id='Layer_1' data-name='Layer 1' xmlns='http://www.w3.org/2000/svg' viewBox='0 0 1920 1200'%3E%3Cdefs%3E%3Cstyle%3E.cls-1%7Bopacity:0.1;%7D.cls-2%7Bfill:%230092ff;%7D.cls-3%7Bfill:%2369c9ff;%7D.cls-4%7Bfill:%23ff8d00;%7D.cls-5%7Bfill:%23ffc280;%7D%3C/style%3E%3C/defs%3E%3Cg class='cls-1'%3E%3Cpolygon class='cls-2' points='462.45 210.51 386.45 390.14 2.85 0 251.95 0.01 462.45 210.51'/%3E%3Cpolygon class='cls-3' points='0 553.45 0 356.53 250.57 251.95 386.45 390.14 0 553.45'/%3E%3C/g%3E%3Cg class='cls-1'%3E%3Cpolygon class='cls-4' points='1457.55 989.49 1533.55 809.86 1917.15 1200 1668.05 1199.99 1457.55 989.49'/%3E%3Cpolygon class='cls-5' points='1920 646.55 1920 843.47 1669.43 948.05 1533.55 809.86 1920 646.55'/%3E%3C/g%3E%3C/svg%3E");
```
Comment (jflheureux) [inline Website/src/rendering/src/assets/css/components/Page Content/ThreeColumnsSection.css:4]: Can this be converted to a Tailwind theme custom backgroundImage?
https://tailwindcss.com/docs/background-image#using-custom-values

```
module.exports = {
  theme: {
    extend: {
      backgroundImage: {
        'pattern-light': "url(...)",
      }
    }
  }
}
```

We might have to escape some quotes/double quotes in the value.

And use it like `@apply bg-pattern-light;`

Same thing for pattern-dark.

**Review by jflheureux — CHANGES_REQUESTED** [review]

File: Website/src/rendering/src/assets/css/components/Page Content/ThreeColumnsSection.css (hunk 1 above)
Comment (mintsitecore) [reply Website/src/rendering/src/assets/css/components/Page Content/ThreeColumnsSection.css:4]: we also have this pattern uploaded in dam, if we want to make life complicated and use the public link from there. but I think this is the best way for better performance.

Comment (adoprog) [reply Website/src/rendering/src/assets/css/components/Page Content/ThreeColumnsSection.css]: Testing theme approach now.

Comment (adoprog) [reply Website/src/rendering/src/assets/css/components/Page Content/ThreeColumnsSection.css]: Okay, looks like it works (double quotes escaped with '\')

**Review by jflheureux — APPROVED** [review]

**Pull request merged**